From b5d0736dd2ed55e5bedc56739e03cfa0c1f5b66f Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Wed, 14 Dec 2022 15:12:16 +0100
Subject: [PATCH 0073/1141] dt-bindings: bus: Add STM32MP25x platforms support
 for STM32 System Bus

Adds st,stm32mp15-sys-bus as a valid compatible for a STM32 System Bus.

Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: Iaca7cf1bafdf2d18469d2cbb4a2f56a2a758eb68
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/281101
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Lionel DEBIEVE <lionel.debieve@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/301070
Reviewed-by: Lionel DEBIEVE <lionel.debieve@foss.st.com>
Reviewed-by: Gabriel FERNANDEZ <gabriel.fernandez@foss.st.com>
---
 Documentation/devicetree/bindings/bus/st,sys-bus.yaml | 1 +
 1 file changed, 1 insertion(+)

--- a/Documentation/devicetree/bindings/bus/st,sys-bus.yaml
+++ b/Documentation/devicetree/bindings/bus/st,sys-bus.yaml
@@ -21,6 +21,7 @@ properties:
     enum:
       - st,stm32mp13-sys-bus
       - st,stm32mp15-sys-bus
+      - st,stm32mp25-sys-bus
 
   reg:
     maxItems: 1
